io_scene_nif/modules/collision/collision_export.py

```diff
@@ -195,7 +195,7 @@ def export_collision_helper(self, b_obj, parent_block):
         # n_havok_mat = (b_obj.nifcollision.havok_material, b_obj.nifcollision.skyrim_havok_material)
         # just hard code to the first entry for now until the version transition is settled
         # then get it from material name
-        n_havok_mat = (0, 131151687)
+        n_havok_mat = (0, 131151687)[0]
```

Comment:
@HENDRIX-ZT2 Here is the workaround I introduced. Going to slap a FIXME on this.